### PR TITLE
83082

### PR DESCRIPTION
**Pull request by saikumar9**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [83082](https://devops.dchbx.org/redmine/issues/83082) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | @karashearer5 |

### CODE REVIEW

File: app/views/notices/ivl/ivl_tax_notice.html.erb [hunk 1]
```diff
@@ -137,7 +137,7 @@
...
 
 <%= render partial: 'notices/shared/paragraph', :locals => {
-content: "Federal law required most Americans to have a minimum level of health coverage or pay a tax penalty through #{previous_year}. The #{Settings.aca.state_name} now has an individual responsibility requirement. This means that DC residents must have qualifying health coverage all year. If you don't have coverage, and don't qualify for an exemption, you’ll have to pay a penalty when you file your DC taxes. For more information: <a href= #{Settings.site.ivl_responsibility_url}>https://dchealthlink.com/individual-responsibility-requirement</a>."} %>
+content: "Federal law required most Americans to have a minimum level of health coverage or pay a tax penalty through 2018. The #{Settings.aca.state_name} now has an individual responsibility requirement. This means that DC residents must have qualifying health coverage all year. If you don't have coverage, and don't qualify for an exemption, you’ll have to pay a penalty when you file your DC taxes. For more information: <a href= #{Settings.site.ivl_responsibility_url}>https://dchealthlink.com/individual-responsibility-requirement</a>."} %>
```
Comment (saimekala07) [inline app/views/notices/ivl/ivl_tax_notice.html.erb]: Is there a reason why dynamic year pulling argument changed to hardcoded 2018?

File: script/ivl_tax_cover_letter_notice.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# rails runner script/ivl_tax_cover_letter_notice.rb, Example: rails runner script/ivl_tax_cover_letter_notice.rb -e production
+begin
+  person = Person.all_consumer_roles.first
```
Comment (saimekala07) [inline script/ivl_tax_cover_letter_notice.rb]: why are we picking the first person with consumer role, are supposed to find/loop person that need to receive notice here.

File: app/views/notices/ivl/ivl_tax_notice.html.erb (hunk 1 above)
Comment (saikumar9) [reply app/views/notices/ivl/ivl_tax_notice.html.erb]: The date is not a dynamic number, it is supposed to be 2018.

File: script/ivl_tax_cover_letter_notice.rb (hunk 2 above)
Comment (saikumar9) [reply script/ivl_tax_cover_letter_notice.rb]: This is just a one-time notice pdf per year that we generate on the Enroll side to use it as a template for the 1095As generation on the glue side. We do not generate these on the Production server.

**Pull request merged**